services/networking/loadbalancer/taxonomy.md

```diff
@@ -0,0 +1,67 @@
+| CCC-050335  | Outpost support                                       | Ability to extend cloud-provisioned load balancing services to on-premise or edge locations using dedicated hardware or software appliances (outposts).                                                                                                                                                                       |
+| CCC-050336  | Same VPC network                                      | Can be configured to operate within the same Virtual Private Cloud (VPC) network as the backend instances they are balancing traffic to.                                                                                                                                                                                      |
+| CCC-050337  | Multiple VPC network                                  | Can be configured to handle requests to backend instances in different VPCs.                                                                                                                                                                                                                                                  |
+| CCC-050338  | External access                                       | Can configure public-facing endpoints to handle requests originating from the internet or other external networks.                                                                                                                                                                                                            |
+| CCC-050339  | Session affinity                                      | Can configure subsequent requests from an initial client to be passed to the same target.                                                                                                                                                                                                                                     |
+| CCC-050340  | Long-lived TCP connection                             | Supports connections that remain open for an extended period without teardown and reestablishment.                                                                                                                                                                                                                            |
+| CCC-050341  | Load Balancing to multiple ports on the same instance | Can load balance to multiple ports on the same instance to distribute incoming traffic across different services or applications running on the same backend server.                                                                                                                                                          |
```

Comment:
Is this applicable at all three OSI layers mentioned earlier?

Reply:
Depending on provider, some of these features are layer agnostic and some are layer dependent. I favoured listing it even if it was layer agnostic, because it's still technically a 'load balancing' feature. For AWS, multiple ports is supported for all LBing layers.

Reply:
OK